_docs/resources/index.md

```diff
@@ -1,7 +1,111 @@
 ---
 title: Resources
-layout: categories
+layout: family
 category: Resources
 ---
 
-Resources are materials and information used for reference.
+<section class="static-section" markdown="1">
+
+## Sketch Resources
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol-6 hxCol-xs-12 hxCol-sm-12 hxCol-md-6 hxCol-lg-6" %}
+
+<div class="resource-card">
+  <div class="icon"><img src="{{site.baseurl}}/assets/images/sketch-library.svg" alt="designer UX icon"/>
+    <h4 style="margin-top: 16px; margin-bottom: 16px;">Sketch Library</h4>
+  </div>
+    <p style="margin-bottom: 16px">The Sketch library is the preferred method for creating high fidelity design deliverables in the Helix language. It is installed directly from Sketch and allows all files access to all ui components.</p>
```

Comment:
s/ It is installed directly from Sketch and allows all files access to all ui components. / It is installed directly via Sketch and brings high-fidelity Helix sketch objects into your sketch files.

Reply:
updated.